consensus/src/epoch_manager.rs

```diff
@@ -67,6 +67,10 @@ impl<T: Payload> LivenessStorageData<T> {
     }
 }
 
+// @REVIEW: FWIW, before I read any code I assumed that an "EventProcessor" is global and that it
+// delegates to an EpochManager that is spawned per epoch. If other people are confused could be
+// worth considering the name
```

Comment:
lol, EpochManger is global and spawns EventProcessor per epoch, suggestions are welcome!

Reply:
Honestly, I'd reverse the two names. EventProcessor sounds like the main() loop for consensus events, EpochManager sounds like it handles a single epoch. But that's just what's intuitive to me...

Anyways, not a big deal, but just want to point out the confusion a first time reader had